spec/tests/angularComponent/ngcOmniboxControllerSpec.js

```diff
@@ -446,6 +446,24 @@ describe('ngcOmnibox.angularComponent.ngcOmniboxController', () => {
       expect(omniboxController.highlightedChoice).toBe(null);
     });
 
+    it('should highlight a specific choice', () => {
+      expect(omniboxController.highlightedChoice).toBe(null);
+
+      omniboxController.highlightChoice('two');
+      expect(omniboxController.highlightedChoice).toBe('two');
+    });
+
+    it('should only highlight a choice that exists in the list of choices', () => {
+      omniboxController.highlightChoice('two');
+      expect(omniboxController.highlightedChoice).toBe('two');
+
+      omniboxController.highlightChoice('four');
```

Comment:
Maybe change 'four' to 'bogus'? It's not clear to me reading this diff that "four" is not in the list of choices. I have to do a lot of code expanding.